site/write_ups/2022/MetaphorsBlogPost

```diff
@@ -0,0 +1,76 @@
+
+Metaphors are by nature partial; they only explain part of the target concept. Consequently, we choose metaphors with the entailments we think best explain the target concept. When we say that time is a resource, we seek to emphasise how time is finite, but obscure how time never stops, or how different people perceive it. 
+
+The choice of metaphor is dictated not only by context, but also by one’s ideology (or worldview); the metaphor is chosen to match the ideology one wishes to promote. Consequently, metaphors provide a window through which we can understand the dominant ideologies in the world of data science. After looking at tens of data science presentations, promotion material, and learning guides, I identified two key data ideologies, which I represent by their main metaphors.
+
+## Data as unprocessed economic potential/resource 
+
+Example metaphors: “data is the new oil”, “data gold mine”, “data mining”, “knowledge extraction”, “data is the new electricity”
+
+This data ideology, borne of the business world, takes data as material – often as a natural resource like oil – which when first obtained is unsuitable for generating economic value. To turn the ‘raw’ economic potential into actual economic value, data must be processed (e.g., filtered, aggregated, visualised, and modelled), or as Peter Sondergaard, Senior Vice President of tech-hype-cycle consultancy Gartner put it in 2011,
+
+“Information is the oil of the 21st century, and analytics is the combustion agent.” (Gartner, 2011, cited in Haupt, 2016)
+
+The key claims in this data ideology are that data can be processed to obtain information that can be exchanged for financial gain, and that profit generation is necessarily good. It follows from these assertions that we should be recording/creating more data and processing it in order to drive economic growth. This is a clearly capitalist view of data, and its metaphors are typically used by data scientists or managers in sales pitches where they appeal for investment in a data-driven project by emphasising the economic value of data.
+
+By treating data as economic potential/resource, this ideology and its metaphors background the human side of data: data are recorded about humans and are used to make decisions that affect humans. This can have serious negative consequences for the data subjects as witnessed by numerous case studies (Marsan, 2012), such as the introduction of Google’s Street View, which was initially launched without the blurring of faces and vehicle numberplates, leading to people’s privacy being compromised (Shankland, 2008). 
```

Comment:
```suggestion
By treating data as economic potential/resource, this ideology and its metaphors background the human side of data: data are recorded about humans and are used to make decisions that affect humans. 
This can have serious negative consequences for the data subjects as witnessed by numerous case studies (Marsan, 2012), such as the introduction of Google’s Street View, which was initially launched without the blurring of faces and vehicle numberplates, leading to people’s privacy being compromised (Shankland, 2008). 
```